models/classes/LtiProvider/LtiProviderFactory.php

```diff
@@ -91,6 +91,10 @@ public function createFromArray(array $provider): LtiProvider
 
     private function getLtiVersion(array $propertiesValues): string
     {
+        if (empty($propertiesValues[RdfLtiProviderRepository::LTI_VERSION])) {
```

Comment:
can you provide the use case for this ? 
because i get what you did but i don't know when this happens

Reply:
That may happens if RDF instance was created with older code ( not lti1.3 aware ) 
So that can solved either by such if of data migration

Reply:
Yes, this is to be BE compatible :). Thanks for the explanation @boajer